portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -164,13 +170,28 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty(COMMENT_LENGTH, text.length());
     commentEntity.setProperty(COMMENT_NICKNAME, nickname);
     commentEntity.setProperty(COMMENT_IMAGE_URL, imageUrl);
+    commentEntity.setProperty(COMMENT_SENTIMENT, sentiment);
     
     this.datastore.put(commentEntity);
 
     // Redirects to the bottom of the current page to see new comment added.
     response.sendRedirect(BOTTOM_OF_PAGE);
   }
 
+  /** 
+   * Returns a sentiment score based on {@code text}.
```

Comment:
Please add information about what the range of the return value is and what it means.

Reply:
Done!